uc-cdis/gen3-helm
# Initial helm charts for services and add precommit

All these services have been tested and deployed to a local (Docker desktop) kubernetes cluster. 

To test run the following: 

If you have nginx-ingress controller installed in your cluster add the following to your hosts file to be able to hit these servies from a browser: 

```
127.0.0.1 ingress.local
```

To install nginx-ingress run the following command

```
helm upgrade --install ingress-nginx ingress-nginx \
  --repo https://kubernetes.github.io/ingress-nginx \
  --namespace ingress-nginx --create-namespace
```


```
cd helm
helm upgrade --install fence ./fence
helm upgrade --install indexd ./indexd
helm upgrade --install audit ./audit
```

If everything went okay, you should be able to browse to following urls in your browser 

http://ingress.local/index/index
http://ingress.local/user/_status
http://ingress.local/audit/_status
